Library/Formula/gdal.rb

```diff
@@ -308,3 +323,4 @@ def caveats
     system "#{bin}/ogrinfo", "--formats"
   end
 end
+
```

Comment:
I've never noticed that about vi hiding that extra newline before. Opened it up in TextWranger with invisibles shown and yup there was two line feeds. Fixed it now.